addons/rook/template/base/install.sh

```diff
@@ -339,7 +339,57 @@ function rook_cluster_deploy_upgrade() {
 
     rook_cluster_deploy_upgrade_flexvolumes_to_csi
 
-    logSuccess "Rook-ceph cluster upgraded"
+    verify_rook_updated_cluster
+}
+
+# Before to finish end report that the upgrade was done with success ensure that
+# only on rook version is found and the ceph status
+# https://rook.io/docs/rook/v1.10/ceph-upgrade.html#3-verify-the-updated-cluster
+function verify_rook_updated_cluster() {
+    logStep "Checking if Rook Upgrade is completed successfully"
+    echo "Verifying Rook Version Deployed"
+    if ! spinner_until 1200 rook_version_deployed ; then
```

Comment:
Why not use the go code here?

```suggestion
    if ! "$DIR"/bin/kurl rook wait-for-rook-version "$ROOK_VERSION" --timeout=1200 ; then
```

Reply:
I moved to the go code.